app/models/task.py

```diff
@@ -2,4 +2,28 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String, nullable=False)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, default=None)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
```

Comment:
Awesome! 
Fun fact: the default value for nullable is `True` so we don't need to explicitly add it 😊

I believe we should also define the relationship between `Goal` and `Task` here and set it to back populate the records via: 
```
goal = db.relationship("Goal", back_populates="tasks")
```